src/main/java/dream/fcard/gui/controllers/windows/MainWindow.java

```diff
@@ -30,7 +32,7 @@
 public class MainWindow extends VBox {
 
     @FXML
-    private ScrollPane deckScrollPane;
+    private VBox deckScrollPane;
```

Comment:
not sure why this needs to be a vbox also. But if you need to change it, change the name to deckScrollBox instead.

Reply:
okay sure, i’ll edit the names accordingly! didn’t dare to change cos i wasn’t super familiar with the code and didn’t want to break anything, but i think should be ok now